test/functional/fixtures/regression/gh-1057/testcafe-fixtures/index-test.js

```diff
@@ -0,0 +1,21 @@
+import { ClientFunction } from 'testcafe';
+
+fixture `gh-1057`
+    .page `http://localhost:3000/fixtures/regression/gh-1057/pages/index.html`;
+
+const targetsClicked = ClientFunction(() => window.target1Clicked && window.target2Clicked);
+
+test('gh-1057', async t => {
+    // NOTE: scrolling has issues in iOS Simulator https://github.com/DevExpress/testcafe/issues/1237
+    await t
+        .wait(5000)
```

Comment:
if we need wait too much on iOS Simulator, let's do it only in simulator. Use userAgent to determine it